### ISSUE TITLE
Unable to save the schema

### ISSUE THREAD

**Issue by Lovee93** [rpbouman/phase#23]
Hi there, 

Well, firstly I am just a newbie and trying to create the OLAP cube using the same Sakila example. I created the same cube and then I saved it clicking on the save button. Next, I tried to open it in the Saiku Analytics for creating a bar graph or something. But I was unable to find that cube in it. Then I refreshed the Phase window and my Sakila Schema disappeared from the saved schemas. 
So the gist is that while I save any schema, it doesn't appear in the menu, and if I try to create the new schema with the same name, it says it already exists, but I cannot find it. Kindly help me locating the new schemas when they are created.

**Comment by Lovee93** [comment]
So here are the screenshots of the schema(cube) I created following the same steps as you have taught in the video: 

![the schema](https://cloud.githubusercontent.com/assets/6823402/24236326/6bbc65b2-0fc7-11e7-8fc2-d32fa797d0c1.jpg)
![film](https://cloud.githubusercontent.com/assets/6823402/24236285/4b273976-0fc7-11e7-9f99-99b80415362d.png)
![customer](https://cloud.githubusercontent.com/assets/6823402/24236293/4fd15754-0fc7-11e7-98a5-3b52d02abeb6.png)
![film](https://cloud.githubusercontent.com/assets/6823402/24236297/5287da9a-0fc7-11e7-8704-1452eb0f1b94.png)
![staff](https://cloud.githubusercontent.com/assets/6823402/24236303/5525a570-0fc7-11e7-8693-d53e0c21472b.png)
![save schema](https://cloud.githubusercontent.com/assets/6823402/24236306/571804cc-0fc7-11e7-9061-6d3ab553476e.png)

Now after I saved the schema I tried to access it within the Saiku Analytics, but couldn't find it.
![saiku_cube_not_available](https://cloud.githubusercontent.com/assets/6823402/24236308/59201ee4-0fc7-11e7-93cd-80230f9f6652.png)

Next, I refreshed the Phase window with the green refresh button and the Sakila schema disappeared from the list.
![on_refresh_schema_disappeared](https://cloud.githubusercontent.com/assets/6823402/24236313/5ae114cc-0fc7-11e7-84ba-f8e2390a9cd5.png)

Then I tried to create a new schema with the same name Sakila and it showed me the following error.  

![schema_not_there_but_error](https://cloud.githubusercontent.com/assets/6823402/24236329/6e27a78a-0fc7-11e7-89e7-cb1b0ee1a48c.png)

So, I don't understand if the schema was saved? And if it was, then where and how can I actually access it?

Here is the schema file of Sakila:
[Sakila Schema FIle.txt](https://github.com/rpbouman/phase/files/863921/Sakila.Schema.FIle.txt)


Here are the text version of log files: 

[localhost_access_log.2017-03-23.txt](https://github.com/rpbouman/phase/files/863912/localhost_access_log.2017-03-23.txt)

[Catalina_out.txt](https://github.com/rpbouman/phase/files/863923/Catalina_out.txt)

[localhost.2017-03-23_log.txt](https://github.com/rpbouman/phase/files/863925/localhost.2017-03-23_log.txt)

[pentaho_log.txt](https://github.com/rpbouman/phase/files/863926/pentaho_log.txt)

[catalina.2017-03-23_log.txt](https://github.com/rpbouman/phase/files/863928/catalina.2017-03-23_log.txt)

**Comment by Lovee93** [comment]
Kindly help me to resolve the issue.

**Comment by obi-juan1971** [comment]
Hi, I have the exactly same issue. Did you manage to solve it? thanks in advance for any help.

**Comment by Lovee93** [comment]
No. I couldn't resolve it. If you find any solution elsewhere, kindly post it.
